server/plugin.go

```diff
@@ -77,8 +77,6 @@ type Plugin struct {
 }
 
 func (p *Plugin) OnActivate() error {
-	p.client = pluginapi.NewClient(p.API, p.Driver)
```

Comment:
I'm not quite sure if it's really needed, but should we add a 
```golang
	if p.client == nil {
		p.client = pluginapi.NewClient(p.API, p.Driver)
	}
```
just for additional protection against races?

Reply:
Sure. Handled the race condition. Thanks.